app/static/scss/lib/jquery.timepicker.scss

```diff
@@ -1,4 +1,4 @@
-@import '../colors';
+@import '../admin/colors';
```

Comment:
Yeah I don't know why this has been broken for so long. We should def merge this in.